src/main/java/com/overops/plugins/service/impl/RenderService.java

```diff
@@ -80,13 +80,13 @@ public void render() {
         if (getCheckTotalErrors() || getCheckUniqueErrors()) {
             if (getCheckTotalErrors() && getPassedTotalErrorGate()) {
                 outputStream.printStatementHeaders(getTotalErrorSummary());
-            } else if (getCheckTotalErrors() && getPassedTotalErrorGate()) {
+            } else if (getCheckTotalErrors() && !getPassedTotalErrorGate()) {
                 outputStream.printStatementHeaders(getTotalErrorSummary());
             }
```

Comment:
Can we simplify this logic?

```
if (getCheckTotalErrors()) {
  outputStream.printStatementHeaders(getTotalErrorSummary());
}
```

Reply:
Yes, we can, I didn't start refactoring the code.
Do we I need to refactor this class

Reply:
Yes, you clean it up. In this case, since both if and else do the same thing it's clear we can simplify the logic.